english/forest/shelter/shelter.md

```diff
@@ -1,4 +1,11 @@
 You break down some branches and make a frame. You put your parachute over
-the top of it, tying it down securely with palm leaves. Luckily you did it 
-because as you make yourself comfortable for the night the heaven's open. 
-There's a terrible storm brewing... 
\ No newline at end of file
+the top of it, tying it down securely with palm leaves. Luckily you did it
+because as you make yourself comfortable for the night the heaven's open.
+There's a terrible storm brewing...
+
+You see a flash of lightning a clap of thunder. Suddenly you are in the middle of a monsoon.
+The wind blows away your parachute like a dry leaf and buckets of water rain down on you.
+Shivering in the dark wetness you can't believe this is happening.
+Suddenly - [you wake up!](english/wake-up)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.